configure.ac

```diff
@@ -168,6 +168,18 @@ AS_IF([test "x$enable_abootsettings" != xno],
 AM_CONDITIONAL([WANT_ABOOTSETTINGS], [test x$enable_abootsettings != xno])
 
 
+#
+# Wheell is priveleged
+#
+AC_ARG_ENABLE([usewheel],
+  [AS_HELP_STRING([--enable-usewheel],
+    [use wheel group users as priveleged])])
```

Comment:
I'm wondering if that explanation could be clarified a bit?

Say, something like "Users in wheel group are considered privileged", or maybe "Only users..." depending on whether root user that does not belong to wheel group is intentionally excluded or not.